server/core/src/https/views/mod.rs

```diff
@@ -8,8 +8,12 @@ use axum::{
 
 use axum_htmx::HxRequestGuardLayer;
 
+use concread::cowcell::CowCellReadTxn;
```

Comment:
There is an exposed type for this https://github.com/kanidm/kanidm/blob/master/server/core/src/https/extractors/mod.rs#L18 which already has the concread wrapper, so you don't need concread as a dep in server/views here at all.